monitored-httpclient/src/test/java/com/github/nhenneaux/resilienthttpclient/monitoredclientpool/HttpClientPoolTest.java

```diff
@@ -161,7 +161,7 @@ void shouldReturnToString() {
                 .build()
         ) {
             assertFalse(httpClientPool.getNextHttpClient().isEmpty());
-            assertThat(httpClientPool.check().getDetails().toString(), stringContainsInOrder("[ConnectionDetail{hostname='google.com', inetAddress=google.com/", ", healthUri=https://", ":443, healthy=true}, ConnectionDetail{hostname='google.com', inetAddress=google.com/", ", healthUri=https://", ":443, healthy="));
+            assertThat(httpClientPool.check().getDetails().toString(), stringContainsInOrder("[ConnectionDetail{hostname='google.com', hostAddress=", ", healthUri=https://", ":443, healthy=true}, ConnectionDetail{hostname='google.com', hostAddress=", ", healthUri=https://", ":443, healthy="));
```

Comment:
And why is `hostAddress` empty here?

Reply:
it's not empty but it's a dynamic address so I prefer to avoid comparing it inside the test